flink-autoscaler/src/main/java/org/apache/flink/autoscaler/ParallelismAdjuster.java

```diff
@@ -100,4 +109,28 @@ private static int calculateMinimumParallelism(
         p = Math.max(p, parallelismLowerLimit);
         return p;
     }
+
+    /** The mode of the key group or parallelism adjustment. */
+    public enum KeyGroupOrPartitionsAdjustMode implements DescribedEnum {
+        ABSOLUTELY_EVENLY_DISTRIBUTION(
```

Comment:
> The name is confusing because this doesn't guarantee absolutely even distribution.
> 
> How about `EVENLY_SPREAD`?

Looks like `EVENLY_SPREAD` is more reasonable, I'm fine with that
@1996fanrui What do you think about this ?

Reply:
Sounds good to me.